protobuf/protoc-gen-govalidator/plugin/plugin.go

```diff
@@ -256,6 +257,10 @@ func genErrorString(g *protogen.GeneratedFile, varName, fieldName, specificErr s
 	g.P(`return `, fmtPackage.Ident("Errorf"), "(`", fieldName, `: value '%v' must `, specificErr, "`, ", varName, `)`)
 }
 
+func genErrorStringWithParams(g *protogen.GeneratedFile, varName, fieldName, specificErr string, errParams []string) {
```

Comment:
I think `errParams` would be nicer API as a variadic  function argument ie. `errParams ...string`

Reply:
good idea